src/network/network_content.cpp

```diff
@@ -70,9 +70,9 @@ bool ClientNetworkContentSocketHandler::Receive_SERVER_INFO(Packet *p)
 	ci->dependencies = MallocT<ContentID>(ci->dependency_count);
 	for (uint i = 0; i < ci->dependency_count; i++) ci->dependencies[i] = (ContentID)p->Recv_uint32();
 
-	ci->tag_count = p->Recv_uint8();
-	ci->tags = MallocT<char[32]>(ci->tag_count);
-	for (uint i = 0; i < ci->tag_count; i++) p->Recv_string(ci->tags[i], lengthof(*ci->tags));
+	uint tag_count = p->Recv_uint8();
+	ci->tags.reserve(tag_count);
+	for (uint i = 0; i < tag_count; i++) ci->tags.push_back(p->Recv_string(32));
```

Comment:
would you mind making `32` into an enum value, like the rest, while here? Would be appreciated :)

Reply:
I have been pondering doing that, but then I'd "need" to add them to the massive list of "enum" values for the other sizes of things in the network and only use them once. 
So I was somewhat in doubt whether that actually makes things much clearer. As I would replace the 32 here with NETWORK_CONTENT_TAG_LENGTH, but that could already be inferred by it reading a string from the packet into the tag list of content.

Reply:
Well, in general magic numbers are bad ofc, I think it is more a question if we want to address that here and now :) Even adding it to the big list and using it only once has benefits: it is clear that the length is only used in one place. So if someone extends it to a higher number, it is very clear that there are no lingering problems because you missed a "32 - 1" situation etc :D

So if you don't mind, I would really appreciate it, but I also fully understand if it is "future work" :)